src/style/properties.js

```diff
@@ -35,6 +35,8 @@ const styfn = {};
     numbers: { number: true, unitless: true, multiple: true },
     positiveNumber: { number: true, unitless: true, min: 0, strictMin: true },
     size: { number: true, min: 0 },
+    dashStyle: { number: true, min: 0, multiple: true },
+    lineDashOffset: { number: true },
```

Comment:
I don't think these types are necessary.  `number` and `numbers` should suffice.    Future rendering implementations might not allow for with-units values.  And I remember some browsers in the past not yielding precise results, so it's probably best to just consider the values to be unitless and relative.